metrics/pearsonr/pearsonr.py

```diff
@@ -94,5 +94,5 @@ def _info(self):
 
     def _compute(self, predictions, references):
         return {
-            "pearsonr": pearsonr(references, predictions)[0],
+            "pearsonr": pearsonr(references, predictions)[0].tolist(),
```

Comment:
i just discovered in #2613 that `numpy.ndarray.item` might be a closer fit to what we're trying to achieve and also less confusing to future developers of the library :)

Reply:
I switched to using .item() here as well, thanks !